src/ui/windows/TogglDesktop/TogglDesktop/ui/windows/MainWindow.xaml.cs

```diff
@@ -121,6 +126,9 @@ private void initializeContextMenu()
                     asMenuItem.CommandTarget = this;
                 }
             }
+
+            ShowMainWindowCommand = ReactiveCommand.Create(this.ShowOnTop, mainContextMenu.WhenAnyValue(x => x.PlacementTarget)

```

Comment:
Discovered a bug: this item also appears on the tray icon menu, if the context menu was previously open on the MiniTimer.
Proposing an, admittedly hacky, fix: also check if contextMenu.Placement != Placement.AbsolutePoint, because this is what tray icon context menu is using (probably a hack in itself from `Hardcodet.Wpf.TaskbarNotification` library, but at least we can say it's not applicable for other appearances of this context menu).

Reply:
In this case I'd rather vote for different xaml for mini timer. It shouldn't be that much work. I'll try how it'd look and work.